.clang-tidy

```diff
@@ -49,7 +49,7 @@ modernize-*,
 performance-*,
 readability-container-size-empty,
 '
-HeaderFilterRegex: '^(c10/|torch/csrc/).*$'
+HeaderFilterRegex: '^(?!build).*$'
```

Comment:
I don't think clang-tidy's regex actually supports negative lookaheads unfortunately, so that makes this regex a bit difficult to construct.

Reply:
Done